packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+#### Client-based isolation for users accessing `/authorize` from a NAT IP
+
+Alice, Bob, and Lisa all work from the same office. Since they access Okta through a Network Address Translation (NAT) IP from their office network, everyone shares the same IP address. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limiting framework creates a unique per minute request quota for the combination of every user's IP address, OAuth client ID of the application, and the device identifier set by Okta in each user's browser.
+
+![Client-based isolation for users accessing the authorize endpoint from a NAT IP](/img/diagram2.png "Displays Alice as portal123 Device1, Bob as portal123 Device2, Lisa as portal123 Device3, all making authorize requests that are a combination of the shared IP address")
+
+* Alice: (NAT IP + portal123 + Device1 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* Bob: (NAT IP + portal123 + Device2 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* Lisa: (NAT IP + portal123 + Device3 ID) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+
+> **Note:** The device identifier is derived from a cookie (`dt` cookie) that Okta sets in the browser when the first request is made to Okta. When the requests are made using a non-browser client, the device cookie isn't present. In that case, such requests fall under a common quota with the device identifier being null (NAT IP + portal123 + null).
+
+#### Client-based isolation for users accessing `/authorize` through a proxy
+
+When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+
+### Client-based rate-limiting modes
+
+The client-based rate-limiting framework can exist in one of three modes:
+
+| Mode                                     | Description                                                                                                          |
+| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| **Enforce and log per client (recommended)** | Rate limiting is based on client-based rate-limiting values. Client-specific rate limiting violation information is logged by firing System Log events. |
+| **Log per client**                          | Rate limiting is still based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values, but client-specific rate limiting violation information is logged by firing System Log events. |
```

Comment:
Rate limiting is based on the [org-wide Rate Limit] (I don't think we should have "still" in there as it wasn't mentioned before)
client-specific rate limiting violation information is logged **as** System Log events.